tests/test_validate_outputs.py

```diff
@@ -77,27 +77,27 @@ def test_calexp(self):
         # running the test to determine the updated values.
         standard_atol = 5e-7
         for name, var, val, atol in [
-                ("im_mean", im_mean, 4.3885845565891, standard_atol),
-                ("im_std", im_std, 163.46922027517536, standard_atol),
-                ("var_mean", var_mean, 51.764979094464934, standard_atol),
-                ("var_std", var_std, 48.19498276625069, standard_atol),
-                ("num_good_pix", num_good_pix, 7725755.00000000000000, 0),
-                ("psf_ixx", psf_ixx, 4.253191896391297, 6e-7),
-                ("psf_iyy", psf_iyy, 4.687397483153177, 7e-7),
-                ("psf_ixy", psf_ixy, -0.57911628487574, standard_atol),
-                ("summary.psfSigma", summary.psfSigma, 2.11203591780044, standard_atol),
-                ("summary.psfIxx", summary.psfIxx, 4.272794013403168, 5.1e-7),
-                ("summary.psfIyy", summary.psfIyy, 4.735316824053334, standard_atol),
-                ("summary.psfIxy", summary.psfIxy, -0.57899030354606, standard_atol),
+                ("im_mean", im_mean, 9.573747386158793, standard_atol),
```

Comment:
These first four are huge changes... what happened?

Reply:
See my note in the commit message (seemed like the best place to put that): these images are calibrated to nJy, with a calibration factor of ~2.